### ISSUE TITLE
Server creation example:

### ISSUE THREAD

**Issue by 4815162342lost** [zubkov-andrei/pg_profile#20]
https://github.com/zubkov-andrei/pg_profile/blob/0.3.3/doc/pg_profile.md in doc i found:
```
Server creation example:
SELECT profile.server_new('omega','host=name_or_ip dbname=postgres port=5432');
```

But this function does not exist... Maybe we need create_server instead?

**Comment by zubkov-andrei** [comment]
Yes, example is incorrect, I'll fix it. You should use create_server() function.

**Comment by zubkov-andrei** [comment]
Fixed in 0.3.4
